### DISCUSSION TITLE
Kind of out of this repo question if you please...

### DISCUSSION THREAD

**Discussion by E2GO** [mashb1t/Fooocus#27] [General]
@mashb1t Hello, good sir! Have a question regarding Foocus and Forge future development that may be you can answer. There are not much updates this UI's getting from the creator lllyasviel. There's a lot of talks and rumors going that projects mostly abandoned. As a person who is more close to lllyasviel then everybody else can you please spread some light on whether it true or not? Of course if you have any not "private" information to tell) 
Best regards and any answer appreciated!

**Comment by mashb1t** [comment]
Hey @E2GO, short version: i have no idea.
Contrary to public belief, i am in no way associated with  lllyasviel or have anything to do with him, but i'm a contributor/maintainer of Fooocus and AI enthusiast myself. There have not been contributions by him since December afaik and i don't know about the project roadmap on forge, but he mentioned that he will be back when the next version of SD releases, so hopefully on release of SD3.
Maybe there is also soon going to be the release of a surprise project as lllyasviel has contributed many commits to a private project. We'll see!
Thank you for staying curious and staying tuned, i'm honestly as curious as you are!

**Reply by E2GO** [reply]
Thank you for fast answer and really appreciate for great work on supporting Foocus - it worth it)
